### PR TITLE
Challenge flows

### PR DESCRIPTION
**Pull request by Aleechetorres**
All instructions in step 8 "Challenge Flows" were carried out

A solution to the blockage is given with the suggestion of @abdulflorez

### CODE REVIEW

File: cypress/e2e/google.cy.ts [hunk 1]
```diff
@@ -1,6 +1,84 @@
...
-  it("should have a title", () => {
-      cy.visit("https://www.google.com/");
-      cy.title().should("eq", "Google");
+describe("Log in feature", () => {
+  beforeEach(() => {
+    cy.visit("https://www.demoblaze.com/index.html");
+    
+    // Realizar el login a la página
+    cy.get("#login2").click();
+    //cy.wait(500);
```
Comment (DanielCalleCO) [inline cypress/e2e/google.cy.ts]: Borra esta línea comentada por favor.

File: cypress/e2e/google.cy.ts [hunk 2]
```diff
@@ -1,6 +1,84 @@
...
+    cy.get("#nameofuser").should('contain.text', "Welcome aleechetorres");
   });
-});
\ No newline at end of file
+
+  it("Añadir celular al carrito y validar que el nombre corresponde al item seleccionado", () => {
+    cy.get(".card-title").contains("Iphone 6 32gb").click();
+    cy.get("#tbodyid").should("contain", "Iphone 6 32gb");
+    cy.get(".btn-lg").contains("Add to cart").click();
+    //cy.wait(500);
```
Comment (DanielCalleCO) [inline cypress/e2e/google.cy.ts]: Borra este comment

File: cypress/e2e/google.cy.ts [hunk 3]
```diff
@@ -1,6 +1,84 @@
...
+  });
+
+  it("Añadir Laptop al carrito y validar que el nombre corresponde al item seleccionado", () => {
+    cy.get(".nav-link").contains("Home").click();
+    cy.get(".list-group-item").contains("Laptops").click();
+    cy.get(".card-title").contains("MacBook Pro").click();
+    cy.get("#tbodyid").should("contain", "MacBook Pro");
+    cy.get(".btn-lg").contains("Add to cart").click();
+    //cy.wait(500);
```
Comment (DanielCalleCO) [inline cypress/e2e/google.cy.ts]: Borra el comment

File: cypress/e2e/google.cy.ts [hunk 4]
```diff
@@ -1,6 +1,84 @@
...
+  });
+
+  it("Añadir Monitor al carrito y validar que el nombre corresponde al item seleccionado", () => {
+    cy.get(".nav-link").contains("Home").click();
+    cy.get(".list-group-item").contains("Monitors").click();
+    cy.get(".card-title").contains("Apple monitor 24").click();
+    cy.get("#tbodyid").should("contain", "Apple monitor 24");
+    cy.get(".btn-lg").contains("Add to cart").click();
+    //cy.wait(500);
```
Comment (DanielCalleCO) [inline cypress/e2e/google.cy.ts]: Borrar

File: cypress/e2e/google.cy.ts [hunk 5]
```diff
@@ -1,6 +1,84 @@
...
+      // Hacer la validación después de la iteración
+      cy.get("#totalp").should("contain", suma);
+    });
+  });
+  
+  
+
+  it("Completar la compra", () =>{
+    cy.get(".nav-link").contains("Cart").click();
+    //cy.wait(2000);
```
Comment (DanielCalleCO) [inline cypress/e2e/google.cy.ts]: Delete this

File: cypress/e2e/google.cy.ts [hunk 6]
```diff
@@ -1,6 +1,84 @@
...
+    cy.get(".btn-success").contains("Place Order").click();
+    cy.get("#name").type('Alejandra Echeverri');
+    cy.get("#country").type('Colombia');
+    cy.get("#city").type('Manizales');
+    cy.get("#card").type('3029-9876-4653');
+    cy.get("#month").type('06');
+    cy.get("#year").type('2026');
+    cy.get(".btn-primary").contains("Purchase").click();
+    //cy.wait(1500);
```
Comment (DanielCalleCO) [inline cypress/e2e/google.cy.ts]: Delete

File: cypress/e2e/google.cy.ts [hunk 7]
```diff
@@ -1,6 +1,84 @@
...
+    cy.get(".btn-primary").contains("OK").click();
+  })
+
+  it("Enviar un mensaje a través de Contact", () => {
+    cy.get(".nav-link").contains("Contact").click();
+    //cy.wait(2000);
```
Comment (DanielCalleCO) [inline cypress/e2e/google.cy.ts]: Delete

**Review by DanielCalleCO — CHANGES_REQUESTED** [review]

File: cypress/e2e/google.cy.ts (hunk 1 above)
Comment (Aleechetorres) [reply cypress/e2e/google.cy.ts]: Hecho dani!

**Review by DanielCalleCO — APPROVED** [review]

File: cypress/e2e/google.cy.ts (hunk 3 above)
Comment (Aleechetorres) [reply cypress/e2e/google.cy.ts]: Hecho dani!

File: cypress/e2e/google.cy.ts (hunk 4 above)
Comment (Aleechetorres) [reply cypress/e2e/google.cy.ts]: Hecho dani!

File: cypress/e2e/google.cy.ts (hunk 6 above)
Comment (Aleechetorres) [reply cypress/e2e/google.cy.ts]: Hecho dani!

File: cypress/e2e/google.cy.ts (hunk 7 above)
Comment (Aleechetorres) [reply cypress/e2e/google.cy.ts]: Hecho dani!

File: cypress/e2e/google.cy.ts (hunk 5 above)
Comment (Aleechetorres) [reply cypress/e2e/google.cy.ts]: hecho!

**Pull request merged**